nbs/tsdataset.ipynb

```diff
@@ -54,13 +54,15 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #63.            # Define indexes if not given and then extract static features
</pre><p>I think this can be moved to the base class as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>